lib/orderbook/OrderBook.ts

```diff
@@ -341,14 +345,22 @@ class OrderBook extends EventEmitter {
       };
     }
 
-    if (!this.nosanitychecks && this.swaps) {
-      // check if sufficient outbound channel capacity exists
+    if (this.swaps) {
       const { outboundCurrency, outboundAmount } = Swaps.calculateInboundOutboundAmounts(order.quantity, order.price, order.isBuy, order.pairId);
       const swapClient = this.swaps.swapClientManager.get(outboundCurrency);
+
+      // check if client exists
       if (!swapClient) {
```

Comment:
Let's name this `outboundSwapClient` and also do a similar check for `inboundSwapClient`.